tests/testthat/test-simulate.R

```diff
@@ -379,3 +379,42 @@ test_that("simulate_chain_stats is numerically correct", {
     c(1.00, 3.00)
   )
 })
+
+test_that("simulate_scenarios work as expected", {
+  df <- simulate_scenarios(
+    offspring_dist = c("rpois", "rnbinom"),
+    statistic = c("size", "length"),
+    R_seq = seq(0.1, 0.5, 0.1),
+    k_seq = seq(0.1, 0.5, 0.1),
```

Comment:
To reduce test runtimes
```suggestion
    R_seq = seq(0.1, 0.2, 0.1),
    k_seq = seq(0.1, 0.2, 0.1),
```